diagnostic.md

```diff
@@ -7,40 +7,51 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+https://github.com/donpowers/git-diagnostic
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+you clone it
```

Comment:
Here we were looking for what you would do after you have already cloned the repo down and you wanted to start making changes to the files. The first thing you should do is to create a new branch to work on so you don't alter the master branch until you know the code is ready and working.